frame/contracts/src/weights.rs

```diff
@@ -717,299 +719,299 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	}
 	fn instr_call_indirect(r: u32, ) -> Weight {
-		(43_948_000 as Weight)
-			// Standard Error: 31_000
-			.saturating_add((30_226_000 as Weight).saturating_mul(r as Weight))
+		(45_523_000 as Weight)
+			// Standard Error: 27_000
+			.saturating_add((31_622_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_call_indirect_per_param(p: u32, ) -> Weight {
-		(78_722_000 as Weight)
-			// Standard Error: 5_000
-			.saturating_add((1_199_000 as Weight).saturating_mul(p as Weight))
+		(79_764_000 as Weight)
+			// Standard Error: 6_000
+			.saturating_add((1_198_000 as Weight).saturating_mul(p as Weight))
 	}
 	fn instr_local_get(r: u32, ) -> Weight {
-		(48_083_000 as Weight)
-			// Standard Error: 12_000
-			.saturating_add((805_000 as Weight).saturating_mul(r as Weight))
+		(46_510_000 as Weight)
+			// Standard Error: 11_000
+			.saturating_add((810_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_local_set(r: u32, ) -> Weight {
-		(47_710_000 as Weight)
+		(46_567_000 as Weight)
 			// Standard Error: 11_000
-			.saturating_add((843_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add((833_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_local_tee(r: u32, ) -> Weight {
-		(44_785_000 as Weight)
-			// Standard Error: 11_000
-			.saturating_add((1_466_000 as Weight).saturating_mul(r as Weight))
+		(43_439_000 as Weight)
+			// Standard Error: 10_000
+			.saturating_add((1_463_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_global_get(r: u32, ) -> Weight {
-		(50_106_000 as Weight)
-			// Standard Error: 20_000
-			.saturating_add((1_641_000 as Weight).saturating_mul(r as Weight))
+		(46_201_000 as Weight)
+			// Standard Error: 19_000
+			.saturating_add((1_635_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_global_set(r: u32, ) -> Weight {
-		(47_058_000 as Weight)
+		(42_639_000 as Weight)
 			// Standard Error: 18_000
-			.saturating_add((1_707_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add((1_652_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_memory_current(r: u32, ) -> Weight {
-		(51_494_000 as Weight)
-			// Standard Error: 15_000
-			.saturating_add((925_000 as Weight).saturating_mul(r as Weight))
+		(51_159_000 as Weight)
+			// Standard Error: 14_000
+			.saturating_add((917_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_memory_grow(r: u32, ) -> Weight {
-		(36_983_000 as Weight)
-			// Standard Error: 3_602_000
-			.saturating_add((634_259_000 as Weight).saturating_mul(r as Weight))
+		(36_462_000 as Weight)
+			// Standard Error: 867_000
+			.saturating_add((187_724_000 as Weight).saturating_mul(r as Weight))
 	}
 	fn instr_i64clz(r: u32, ) -> Weight {
-		(60_711_000 as Weight)
-			// Standard Error: 22_000
-			.saturating_add((1_157_000 as Weight).saturating_mul(r as Weight))
+		(53_776_000 as Weight)
+			// Standard Error: 11_000
+			.saturating_add((1_307_000 as Weight).saturating_mul(r as Weight))
```

Comment:
Those instruction benchmarks seem to be less affected. This is just inaccuracy of measurement. I would say this is fine.

Reply:
Thanks for your comments! These help me to understand what is going on :smile: